src/test/three-components/TextureUtils-spec.ts

```diff
@@ -130,57 +135,34 @@ suite('TextureUtils', () => {
           textures =
               await textureUtils.generateEnvironmentMapAndSkybox(HDR_EQUI_URL);
           expect((textures.skybox!.texture as any).isTexture).to.be.ok;
-          expect((textures.environmentMap as any).isTexture).to.be.ok;
+          expect((textures.environmentMap.texture as any).isTexture).to.be.ok;
 
           expect(textureMatchesMeta(textures.skybox!.texture, {
             mapping: 'Cube',
             url: HDR_EQUI_URL
           })).to.be.ok;
 
-          expect(textureMatchesMeta(textures.environmentMap, {
-            mapping: 'Cube',
+          expect(textureMatchesMeta(textures.environmentMap.texture, {
+            mapping: 'PMREM',
             url: HDR_EQUI_URL
           })).to.be.ok;
         });
 
     test(
-        'returns an environmentMap and skybox texture from url with PMREM',
+        'returns an environmentMap and skybox texture from two urls',
```

Comment:
Did we lose a test condition in this cleanup? I think there were two cases tested previously:

 1. A skybox is generated, and an environment map is generated for me from the skybox
 2. A skybox is generated, and an environment map is generated for me from the given URL

Reply:
No, we only lost a case for doing it with/without PMREM. We still have the test above that makes them both from a skybox URL, and we have this one that makes them from separate URLs.